### DISCUSSION TITLE
Check whether `tag_cardinality_limit` transform has acted on an event (metric) in downstream transformers

### DISCUSSION THREAD

**Discussion by ali-sattari** [vectordotdev/vector#17304] [Q&A] [answered]
Hello!

Imagine I am using [tag_cardinality_limit](https://vector.dev/docs/reference/configuration/transforms/tag_cardinality_limit/) transform to curb tags with `drop_tag` action. I want to be able to distinguish between metrics that have been affected by that transform, and those who passed through untouched. Use case would be to log them somewhere, or attach a tag to them to mark this and act on it in destination.

Is there a way to get that info from some metadata using a `remap` transform or something? In other words, do transforms in general, and `tag_cardinality_limit` in particular, leave a mark on the event that they modified it?

**Comment by spencergilbert** [comment, marked as answer]
Hi @ali-sattari - I don't believe we do today, but that does seem like a reasonable enhancement to make. Know the `remap` transform can end up injecting metadata that's similar so there's definitely some existing patterns we can look to.

Would you mind opening a new feature request issue?

**Comment by ali-sattari** [comment]
Thanks @spencergilbert, opened an issue [here](https://github.com/vectordotdev/vector/issues/17321).
